### PR TITLE
Added front-end asset for ss -ntupa

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Trello: https://trello.com/c/q4Yk7iTR
Parser: https://github.com/RedHatInsights/insights-core/pull/1318

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -6307,6 +6312,11 @@
                     "archive_file_name": "/insights_commands/ss_-tulpn",
                     "command": "/usr/sbin/ss -tulpn",
                     "pattern": []
+                },
```
Comment (xiangce) [inline uploader.json]: No need to update the 'Specs' part.

File: uploader.json [hunk 2]
```diff
@@ -644,6 +644,11 @@
             "pattern": [],
             "symbolic_name": "ss"
         },
+        {
+            "command": "/usr/sbin/ss -tulpna",
```
Comment (xiangce) [inline uploader.json]: `ss -tupna`

File: uploader.json (hunk 1 above)
Comment (vishwanathjadhav) [reply uploader.json]: "/insights_commands/ss_-tupna" Source file location will be required,  right?

File: uploader.json (hunk 2 above)
Comment (vishwanathjadhav) [reply uploader.json]: Done

File: uploader.json (hunk 1 above)
Comment (xiangce) [reply uploader.json]: Yes, it can be done by the framework
Btw, `uploader.v2.json` should apply the same update.

Comment (vishwanathjadhav) [reply uploader.json]: Done, removed

**Comment by lhuett** [comment]
This is approved

**Comment by bfahr** [comment]
@SteveHNH @wcmitchell hold off on this one, it was changed after we approved and I have to make sure it is not going to break production.

**Comment by bfahr** [comment]
I've added an interim PR #141 to support the existing production parser.

**Comment by bfahr** [comment]
This one needs to remain pending for now.

**Comment by xiangce** [comment]
@vishwanathjadhav - Just added the 'ss -tulpn' to your patch in https://github.com/vishwanathjadhav/insights-frontend-assets/pull/2, please check and merge it when you are back. Thanks.

**Comment by bfahr** [comment]
@vishwanathjadhav @xiangce the `ss_tupna` spec is also in the `uploader_json` branch and will need to be removed by this PR.  You may need to rebase your fork to see it.

**Comment by bfahr** [comment]
@vishwanathjadhav this PR is approved.  @wcmitchell @SteveHNH this PR needs to go in with https://github.com/RedHatInsights/insights-core/pull/1318 which has already been merged.

**Pull request merged**